src/toil/jobStores/aws/jobStore.py

```diff
@@ -344,6 +344,9 @@ def delete(self, jobStoreID):
 
     def getEmptyFileStoreID(self, jobStoreID=None):
         info = self.FileInfo.create(jobStoreID)
+        with info.uploadStream() as _:
```

Comment:
Perhaps this could use a little more explanation on why it is necessary. Probably fine as is.